numpy/core/arrayprint.py

```diff
@@ -706,6 +706,26 @@ def _extendLine(s, line, word, line_width, next_line_prefix, legacy):
     return s, line
 
 
+def _extendLine_pretty(s, line, word, line_width, next_line_prefix, legacy):
```

Comment:
Would be good if you could add a docstring

Reply:
Did you mean full descripions of all parameters? Or simple note is enough?

Reply:
Done